Formula/dbus.rb

```diff
@@ -30,16 +36,16 @@ def install
     # Fix the TMPDIR to one D-Bus doesn't reject due to odd symbols
     ENV["TMPDIR"] = "/tmp"
 
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--localstatedir=#{var}",
-                          "--sysconfdir=#{etc}",
-                          "--disable-xml-docs",
-                          "--disable-doxygen-docs",
-                          "--enable-launchd",
-                          "--with-launchd-agent-dir=#{prefix}",
-                          "--without-x",
-                          "--disable-tests"
+    system "./autogen.sh", "--disable-dependency-tracking",
```

Comment:
It appears so. If anything, it looks like the stable version (1.10.8 on my machine) made this change too, or at least it is an option for building.